lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -170,13 +222,18 @@ class RenderBlockingResources extends Audit {
       return !canDeferRequest;
     }));
 
+    // Recaulculate estimated time
+    const stackSpecificEstimate = Math.max(...Array.from(
```

Comment:
🚲 🏠   on naming. I don't love the monster I might have created here 😆 

While `computeStackSpecific` is a good label for the function that computes stack-specific overrides, it feels much more out of place as the label for a collection of "mostly the original node timings but some might have stack-specific overrides".

How do you feel about calling all of these `adjustedNodeTimings` or something similar when outside of the `computeStackSpecificTiming` function?

Reply:
Ok makes sense